addons/account/models/chart_template.py

```diff
@@ -785,6 +785,33 @@ def _setup_utility_bank_accounts(self, template_code, company, template_data):
             for company_attr_name, account in zip(accounts_data.keys(), accounts):
                 company[company_attr_name] = account
 
+        # No fields on company
+        if not company.parent_id:
+            accounts_data_no_fields = {
+                'account_journal_payment_debit_account_id': {
+                    'name': _("Outstanding Receipts"),
```

Comment:
Do we plan on using these accounts for other cases?
If not, shouldn't we create them only if `account_accountant` is not installed or to be installed?